package/aistore/client/api.py

```diff
@@ -27,6 +28,27 @@ def __init__(self, url):
     def base_url(self):
         return self._base_url
 
+    def list_buckets(self, provider):
+        url = "{}/buckets".format(self.base_url)
+        params = {}
+        if provider != "":
+            params[URL_PARAM_PROVIDER] = provider
+
+        action = {"action": "list", "value": {}}
+        return requests.get(url=url, headers={"Accept": "application/json"}, json=action, params=params)
+
+    def create_bucket(self, bck_name):
```

Comment:
I would prefer passing the [Bck](https://github.com/NVIDIA/ais-etl/blob/master/package/aistore/client/msg.py#L20) object here instead of a string name. In L42, we can get the provider from `bck.provider`.  

I guess the same makes sense even for destroy.

Reply:
In this case, we have to check bucket provider and return an error when it is not AIS. Though, requiring Bck everywhere makes methods consistent.